tests/dataframe/test_repr_pytest.py

```diff
@@ -241,6 +241,14 @@ def test_num_rows_repr_html(self):
         # Restore default
         pd.set_option("display.show_dimensions", show_dimensions)
 
+    def test_num_rows_repr_html_display_none(self):
+        display = pd.get_option("display.notebook_repr_html")
```

Comment:
Put this whole test in a `try:` after `display = ...` and the "restore default" step within a `finally:`

Reply:
Yes :) there are some more places, I added them too